### ISSUE TITLE
Using Divoom Pixoo 16x16 over Web Bluetooth

### ISSUE THREAD

**Issue by murkle** [xitu/jcode-bluetooth#1]
Hi! Does this library let you use the Pixoo 16x16 over Web Bluetooth?

thanks!

**Comment by akira-cn** [comment]
> Hi! Does this library let you use the Pixoo 16x16 over Web Bluetooth?
> 
> thanks!

No. Because the pixoo 16x16 device only supports the serials port protocol.

**Comment by murkle** [comment]
How about with the new chrome://flags/#enable-bluetooth-spp-in-serial-api flag?

When enabled, Bluetooth Serial Port Profile devices will be enumerated for use with the Serial API. – Mac, Windows, Linux, ChromeOS, Fuchsia, Lacros

**Comment by akira-cn** [comment]
Wow, maybe we can try it.

Chrome doesn't support spp bluetooth device yet when I create this repo.

**Comment by murkle** [comment]
I got this Bluetooth SPP device to connect OK https://github.com/murkle/utils/blob/master/webserial/forceTrainerII.html

Having trouble with the Pixoo though, have you had any luck? Any idea what baud rate it wants?
